src/apps/README.md

```diff
@@ -4,6 +4,9 @@
 
 #### The guide below will show you how to get started developing an app. If you have any questions or need support, please get in touch with us via [Discord](https://discord.gg/6MKAy7gv4P) or [Telegram](https://t.me/pillarxdevelopers).
 
+> [!CAUTION]
+> We do not accept any app submissions that ultimately result in an iframe tag being rendered! This compromises the security of the PillarX ecosystem as the content inside of an iframe can be changed anytime to something deceptive and it is not a chance we are willing to take. Any app submissions that result in this will not be accepted.
```

Comment:
"it is not a chance" => "it is not a risk" maybe?

Reply:
Changed - thanks!